brainframe/api/stubs/base_stub.py

```diff
@@ -271,6 +271,18 @@ def _patch(self, api_url,
 
         return self._send_authorized(request, timeout)
 
+    def _options(self, api_url: str, timeout: float) -> Response:
+        """Sends an OPTIONS request to the given URL, managing authentication and
+        handling errors as necessary.
+
+        :param api_url: The /api/blah/blah to append to the base_url
```

Comment:
> The `api/blah/blah` 
:eyes:

Reply:
We have that beautiful comment in EVERY ENDPOINT and I'm not about to stop now!!